app/javascript/admin/controllers/confirmation_modal_controller.js

```diff
@@ -4,11 +4,30 @@ import { displayErrorAlert, displaySnackbar } from '../messageUtilities';
 const confirmationText = (username) =>
   `My username is @${username} and this action is 100% safe and appropriate.`;
 
+window.addEventListener('load', () => {
+  const params = new URLSearchParams(window.location.search);
+
+  if (params.has('outcome')) {
+    displaySnackbar(params.get('outcome'));
+    setTimeout(() => {
+      window.location.search = '';
+    }, 3000);
+  }
+});
```

Comment:
Just catching up on the chat below about this functionality - it works really well but it does lose us one of the (less well known!) accessibility features of the snackbar:

When a user hovers on the snackbar, the timer to dismiss it is temporarily suspended, e.g. here's me hovering the comment subscription one:

https://user-images.githubusercontent.com/20773163/135121486-7516e6c7-7426-46a1-9909-d8555175c80c.mp4

This gives users with cognitive disabilities, or difficulties reading, a bit of extra time to process what this message is about.

Because we're changing the window.location / reloading the page here, the snackbar always disappears after 3000ms. It's a really tricky problem to solve though 🤔 

I'm wondering if we _need_ to reset the location search params. If we could avoid changing the location here, we'd retain that wee a11y feature. I guess it would look strange in the browser path 🤔 🤔 

Maybe we could leverage local storage? Like, place the message in local storage, and then the param could just be 'outcome', without the actual message encoded there? If we see the 'outcome' param, we pull the string out of local storage, show the snackbar, then clear the local storage?

A lot of question marks and thinking emojis in this comment 😂

Reply:
Thank you for pointing out this a11y issue 🙏🏾 

I will explore the localStorage option 👍🏾